jsonrpc/codec.go

```diff
@@ -169,14 +169,31 @@ func (bnh *BlockNumberOrHash) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// stringToBlockNumberSafe parses string and returns Block Number
+// works similar to stringToBlockNumber
+// but treats empty string or pending block number as a latest
+func stringToBlockNumberSafe(str string) (BlockNumber, error) {
```

Comment:
Curious, is it even valid to have an empty string as a parameter?

Reply:
In some cases yes. And in those cases this func should be called